### DISCUSSION TITLE
Has the VGAUTIL.dsk been updated for saving?

### DISCUSSION THREAD

**Discussion by kwadsten** [markadev/AppleII-VGA#32] [Q&A] [answered]
I assumed (falsely) that when I use option 12 to Quit the config utility it would save settings permanently.  That does not appear to be the case.

I have built the latest "main" branch that has saving ability, but I don't think the .dsk image has been updated with save?

Is this correct?

I see I can do some BASIC pokes to save permanently, but was thinking it would be part of the .dsk utility.

Thanks.

**Comment by aaronsmithdev** [comment]
You're correct - it simply runs the various pokes when a menu option is selected. It does save the slot assignment so you shouldn't have to set the slot each time you run it, but no saving otherwise. And, I made this before Mark added the ability to save with pokes, so that's why it doesn't take advantage of that.

I'll compile the latest firmware (have yet to do so after those updates) and get that added into the disk utility. I'll let you know when I've submitted the PR to Mark and will drop back into this chain with the updated disk file.

**Reply by aaronsmithdev** [reply, marked as answer]
I took a look at the Usage.MD and added the appropriate poke command just before the program exits and changed the final message displayed to indicate such. This is untested, but it should work fine - the changes were straight-forward. I'll add a message that lets you know what the program is doing while it's saving and give it another once-over before submitting the PR, but this should at least give you the functionality until that's done.

[APPLEIIVGAUTIL_BETA.zip](https://github.com/markadev/AppleII-VGA/files/13957605/APPLEIIVGAUTIL_BETA.zip)

Please let me know if you have any issues!

**Reply by markadev** [reply]
Thanks @aaronsmithdev !  I also spent some time over the last weekend to update the config program for saving and also the new font selection.  I just haven't had a chance to make a new release yet but I can at least commit the basic program with changes to the repo.

**Reply by aaronsmithdev** [reply]
Oh, nice! Great to hear, @markadev. I saw you got the release out and took a quick look at the disk file. I'll stand down then! These new changes are great. I still haven't had a chance to really sit down and do the firmware update (been busy restoring an Atari), but I'm really glad to see the poke commands/configuration utility relegated to an "every once in a while" activity. Brilliant changes! These will make life a lot easier. They're very much appreciated!

**Comment by kwadsten** [comment]
That’s great, Thanks for the quick response.  I tried the manual POKEs to save and they worked OK.
